### PR TITLE
[#172582262] Amended postgres doc

### PR DESCRIPTION
**Pull request by malcgds**
What
----

Changed postgres doc as required by [ticket](https://www.pivotaltracker.com/n/projects/1275640/stories/172582262).

How to review
-------------

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. …

Who can review
--------------

Preferably a tech writer.

### CODE REVIEW

File: source/documentation/deploying_services/postgresql.md.erb [hunk 1]
```diff
@@ -309,7 +309,10 @@ To upgrade, you must set up a new service and migrate your app data.
 
 #### Downgrade PostgreSQL service plan
 
-You cannot currently downgrade your service plan.
+You can downgrade your service plan provided that:
+
+1. The amount of storage for the new plan is not less than the existing plan.
```
Comment (cmcnallygds) [inline source/documentation/deploying_services/postgresql.md.erb:314]:
```suggestion
- the amount of storage for the new plan is not less than the existing plan.
```

File: source/documentation/deploying_services/postgresql.md.erb [hunk 2]
```diff
@@ -565,7 +568,7 @@ You can view available plans through the Cloud Foundry command line interface, o
 
 #### Paid plans - PostgreSQL
 
-Some service plans are paid and we will bill you based on your service usage.
+Some service plans are paid and we will bill you based on the size of your plan.
```
Comment (cmcnallygds) [inline source/documentation/deploying_services/postgresql.md.erb:571]:
```suggestion
Some service plans are paid, and we will bill you based on the size of your plan.
```

File: source/documentation/deploying_services/postgresql.md.erb [hunk 3]
```diff
@@ -585,20 +588,12 @@ When you use a high availability plan, Amazon RDS provides a hot standby service
 
 Refer to the [Amazon RDS documentation on the failover process](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.MultiAZ.html#Concepts.MultiAZ.Failover) for more information.
 
-You should test how your app deals with a failover to make sure you are benefiting from the high availability plan. Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to arrange for us to trigger a failover for you.
+You should test how your app deals with a failover to make sure you are benefiting from the high availability plan.
```
Comment (cmcnallygds) [inline source/documentation/deploying_services/postgresql.md.erb:591]:
```suggestion
You should test how your app deals with a failover to make sure you are benefitting from the high availability plan.
```

**Review by cmcnallygds — CHANGES_REQUESTED** [review]
Looks good - have suggested some minor style changes

File: source/documentation/deploying_services/postgresql.md.erb (hunk 3 above)
Comment (cadmiumcat) [reply source/documentation/deploying_services/postgresql.md.erb:591]: Add a link to our instructions on [forcing a failover](https://docs.cloud.service.gov.uk/deploying_services/postgresql/#force-a-failover)?

**Review by cadmiumcat — COMMENTED** [review]
Changes make sense 👍

File: source/documentation/deploying_services/postgresql.md.erb (hunk 1 above)
Comment (malcgds) [reply source/documentation/deploying_services/postgresql.md.erb]: Suggestion accepted.

File: source/documentation/deploying_services/postgresql.md.erb (hunk 2 above)
Comment (malcgds) [reply source/documentation/deploying_services/postgresql.md.erb]: Suggestion accepted.

File: source/documentation/deploying_services/postgresql.md.erb (hunk 3 above)
Comment (malcgds) [reply source/documentation/deploying_services/postgresql.md.erb]: Suggestion accepted.

File: source/documentation/deploying_services/postgresql.md.erb [hunk 4]
```diff
@@ -309,7 +309,10 @@ To upgrade, you must set up a new service and migrate your app data.
 
 #### Downgrade PostgreSQL service plan
 
-You cannot currently downgrade your service plan.
+You can downgrade your service plan provided that:
+
+- the amount of storage for the new plan is not less than the existing plan.
```
Comment (cmcnallygds) [inline source/documentation/deploying_services/postgresql.md.erb:314]:
```suggestion
- the amount of storage for the new plan is not less than the existing plan
```

File: source/documentation/deploying_services/postgresql.md.erb [hunk 5]
```diff
@@ -309,7 +309,10 @@ To upgrade, you must set up a new service and migrate your app data.
...
+- both the existing plan and the new plan have the same major version.
```
Comment (cmcnallygds) [inline source/documentation/deploying_services/postgresql.md.erb:315]:
```suggestion
- both the existing plan and the new plan have the same major version
```

**Review by cmcnallygds — CHANGES_REQUESTED** [review]
Requesting two minor changes (removing full stops from unordered list)

**Review by cmcnallygds — APPROVED** [review]

**Pull request merged**